Example/FinancialConnections Example/FinancialConnectionsUITests/FinancialConnectionsNetworkingUITests.swift

```diff
@@ -260,8 +260,8 @@ final class FinancialConnectionsNetworkingUITests: XCTestCase {
 
         app.fc_nativeBankAccount(name: bankAccountName).tap()
 
-        let accountUpdateRequiredContinueButton = app.buttons["account_update_required_continue_button"]
-        XCTAssertTrue(accountUpdateRequiredContinueButton.waitForExistence(timeout: 1))
+        let accountUpdateRequiredContinueButton = app.buttons["generic_info_primary_button"]
```

Comment:
this test was failing because `account_update_required_continue_button` was not being hit...the "generic info drawer" was being hit due to previous logic